private/buf/bufsync/syncer.go

```diff
@@ -818,44 +687,15 @@ func (s *syncer) printSyncPreparation() {
 		"sync preparation",
 		zap.Any("modulesDirsToSync", s.modulesDirsToIdentityOverrideForSync),
 		zap.Any("commitsTags", s.commitsToTags),
-		zap.Any("branchesModulesToSync", s.branchesToModulesForSync),
-		zap.Any("modulesBranchesSyncPoints", s.modulesToBranchesLastSyncPoints),
-	)
-}
-
-// printCommitsForSync prints syncable commits for a given branch.
-func (s *syncer) printCommitsForSync(branch string, syncableCommits []*syncableCommit) {
-	printableCommits := make([]map[string]string, 0)
-	for _, sCommit := range syncableCommits {
-		var commitModules []string
-		for moduleDir, builtModule := range sCommit.modules {
-			commitModules = append(commitModules, moduleDir+":"+builtModule.ModuleIdentity().IdentityString())
-		}
-		printableCommits = append(printableCommits, map[string]string{
-			sCommit.commit.Hash().Hex(): fmt.Sprintf("(%d)[%s]", len(commitModules), strings.Join(commitModules, ", ")),
-		})
-	}
-	s.logger.Debug(
-		"branch commits to sync",
-		zap.String("branch", branch),
-		zap.Any("commits", printableCommits),
+		zap.Any("branchesModulesToSync", s.modulesDirsToBranchesToIdentities),
+		zap.Any("modulesBranchesSyncPoints", s.modulesToBranchesExpectedSyncPoints),
 	)
 }
 
-// syncableCommit holds the modules that need to be synced in a git commit.
+// syncableCommit holds the built module that need to be synced in a git commit.
 type syncableCommit struct {
-	commit  git.Commit
-	modules map[string]*bufmodulebuild.BuiltModule // moduleDir:builtModule
```

Comment:
A single syncable commit does not contain an array of built modules, but just a single built module (the module dir is implied).